scripted_render_pipeline/exporter/webknossos_exporter.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import pathlib
+
+import cluster_tools
+import webknossos
+
+from . import exporter
+
+DEFAULT_VOXEL_SIZE = 4, 4, 100
+
+
+class Webknossos_Exporter(exporter.Downloader):
+    """exporter from render to webknossos datasets
+
+    downloads datasets from the http render api, then saves the dataset to disk
+    in webknossos format
+
+    location: path to save the webknossos dataset on disk
+        note, when using an existing dataset layers can only be added
+    host: render hostname
+    owner: render owner
+    project: render project
+    downscaling: for testing the render project can be downscaled
+        note, this affects voxel_size
+        defaults to 1 (real size)
+    concurrency: amount of threads to use to download concurrently
+        defaults to exporter.DEFAULT_CONCURRENCY
+    voxel_size: size of the voxels in the webknossos dataset
+        defaults to DEFAULT_VOXEL_SIZE
+    downsample: how many times to downsample the data
+        defaults to 7 (max scale of 128 voxels)
+        set to 0 for no downsampling
+    processes: how many parallel processes to use for downscaling
+        defaults to 8
+    """
+
+    def __init__(
+        self,
+        location,
+        *args,
+        downscaling,
```

Comment:
downscaling is an arg of the parent class, redefining it and then not providing it to the base class causes it to always be the default value